### DISCUSSION TITLE
How to build Storybook 6.0 on Vercel?

### DISCUSSION THREAD

**Discussion by lifeiscontent** [vercel/vercel#4283] [General] [answered]
I can build locally, no problems at all.

I can't figure out how to get now.sh to build my storybook.

I've even put together a test case in my GitHub, and reached out to support team but no one will respond to me.

now.sh is saying webpack loaders are missing for typescript, but they're in the package.json

**Comment by mcsdevv** [comment]
Hey @lifeiscontent, sorry you're having trouble with this! I've looked at the ticket and a reply was last sent to you on Monday, but no response has been received from you since then, unfortunately.

If you can respond to the ticket, we can start making some progress on this for you 👍

**Comment by mcsdevv** [comment]
Also, to provide an answer here, Storybook recommend the following `build` script which Vercel will automatically detect upon import:

```
"build": "build-storybook -c .storybook -o build"
```

**Comment by lifeiscontent** [comment]
@mcsdevv I've just updated the build with that command and it still doesn't work.

**Comment by lifeiscontent** [comment]
ping :)

**Comment by leerob** [comment, marked as answer]
I wrote a guide to make deploying Storybook to Vercel easier. Hopefully this helps, let me know if you have further questions.

https://vercel.com/guides/storybook-with-vercel
